cmd/syncthing/gui_statics.go

```diff
@@ -115,14 +118,32 @@ func (s *staticsServer) serveAsset(w http.ResponseWriter, r *http.Request) {
 		}
 
 		// Check for a compiled in default asset.
-		bs, ok = s.assets[config.DefaultTheme+"/"+file]
+		asset, ok = s.assets[config.DefaultTheme+"/"+file]
 		if !ok {
 			http.NotFound(w, r)
 			return
 		}
 	}
 
+	etag := fmt.Sprintf("%d", asset.Modified.Unix())
+
+	w.Header().Set("Last-Modified", asset.Modified.Format(http.TimeFormat))
```

Comment:
I'd skip this whole `auto.Asset` and `asset.Modified` business and use our `main.BuildStamp` as the modification time for all assets...?

Reply:
But this is nice :(

Reply:
I mean yeah it looks proper and all, pats on the back for that. But the end result is the same with less code. :)

Reply:
I'll have to path up relaypoolsrv somehow tho, so not sure how much less code it will be.

Reply:
Fine.

Reply:
Must be `asset.Modified.UTC().Format(http.TimeFormat)` though or the time stamp might be a lie since the format hard codes `GMT`

Reply:
Or maybe better add the `.UTC()` to when it parses the unixtime at startup.

Reply:
I have a distinct memory we already had all this code at one point and already had the bug with the time zone format and the if-modified-since either always failing or always succeeding depending on which side of the greenwich line you were on... Due to the time.Parse() handling zones correctly and the `.Format(http.TimeFormat)` just being an ugly hack really

Reply:
The format indicates that it should expect a timezone no?

Reply:
https://play.golang.org/p/Did5Tn_OFDQ

Reply:
Yep